community/postsrsd/postsrsd.initd

```diff
@@ -0,0 +1,29 @@
+#!/sbin/openrc-run
+
+description="Postfix SRS Daemon"
+pidfile="/run/postsrsd.pid"
+command="/usr/sbin/postsrsd"
+command_args="-D -d ${SRS_DOMAIN} -s ${SRS_SECRET} -f ${SRS_FORWARD_PORT}
+              -r ${SRS_REVERSE_PORT} -p \"${pidfile}\" -u ${RUN_AS}
+              -c \"${CHROOT}\" -a \"${SRS_SEPARATOR}\""
```

Comment:
You should set sensible defaults in initd file, if possible.
When you define so many custom variables, then you should also add commented confd file.
Please prefer lower case variable names.